README.md

```diff
@@ -39,16 +49,27 @@ HugeNum 900000000000000001.0
 
 Addition is implemented using an elementary-school method. Multiplication follows [Karatsuba](https://en.wikipedia.org/wiki/Karatsuba_algorithm).
 
-## Installation
-
-```
-bower install purescript-precise
-```
+The quick start covers a common, minimal use case for the library, whereas longer examples and tutorials are kept in the [docs directory](./docs.)
```

Comment:
I don't think this is necessary if the quick start is included (as it is here). We could take this line out.

Reply:
Done.